src/undo.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+const readline = require('readline');
+const tty = require('tty');
+const chalk = require('chalk');
+
+const {
+  getConfig,
+  logUndoConfirmation,
+  logUndoConclusion
+} = require('./helpers');
+const {
+  readDirPromise,
+  rimrafPromise,
+  statPromise
+} = require('./utils');
+
+const config = getConfig();
+
+// Delete component directory
+readDirPromise(config.dir).then(files => {
+  if (files.length === 0) {
+    console.log(`There\'s no files in ${config.dir}! Is this where your components
```

Comment:
Nit: I don't think the escaped single-quote is necessary, since you're using backticks.